### DISCUSSION TITLE
Gestion des utlisateurs

### DISCUSSION THREAD

**Discussion by Guerisan** [dbarzin/mercator#226] [Q&A] [answered]
Bonjour Didier,
Lorsque je connecte un utilisateur avec un rôle restreint, il a quand même accès à toutes les fonctionnalités comme un administrateur.
Même chose quand je crée un rôle avec seulement quelques autorisations, il peut quand même tout visualiser/modifier/créer/supprimer.

Le détail des étiquettes apparaissent pourtant bien dans table des rôles.

Est-ce qu'il manque une option à valider quelque part ?

**Comment by dbarzin** [comment, marked as answer]
Merci, il y avait un bug dans la gestion des rôles.
C'est corrigé.
Didier
